templates/volumes.yaml

```diff
@@ -0,0 +1,25 @@
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  name: {{ include "codexdocs.fullname" . }}-db
+  labels:
+    {{- include "codexdocs.labels" . | nindent 4 }}
+spec:
+  accessModes:
+    - ReadWriteOnce
+  resources:
+    requests:
+      storage: 50Mi
```

Comment:
I don't know if it will work on Scaleway, because their block storage starts at size 1Gi

Reply:
let's try